package.json

```diff
@@ -1,25 +1,31 @@
 {
   "name": "route-recognizer",
-  "version": "0.2.8",
+  "version": "0.3.0",
```

Comment:
Since we consider `dist` our public API I think we can do this as a patch version bump.